packages/ember-runtime/lib/system/core_object.js

```diff
@@ -45,241 +45,225 @@ import {
 const applyMixin = Mixin._apply;
 const reopen = Mixin.prototype.reopen;
 
-function makeCtor(base) {
+// should use WeakSet
```

Comment:
Now that the PR moving weakset polyfill into Ember-utils package has landed, I think this can be updated